02-molecules/cards/cards.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Card from './Card/Card.component';
+// import NewsCard from './news/NewsCard.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Molecules/Cards' };
+
+// Component Examples
+export const card = () => (
+  <Card
+    outputImage={true}
+    image={{
+      alt: 'Random Image from Picsum',
+      src: 'https://picsum.photos/400/300.jpg',
+    }}
+    heading="This is the card heading"
+    subheading="Curabitur non nulla sit amet nisl tempus convallis quis ac lectus"
+    body="Ellentesque in ipsum id orci porta dapibus. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a."
+    buttonText="Read more about this card"
+  />
+);
+export const cardBg = () => (
```

Comment:
Let's rename this cardWithBackground to match the story in the twig storybook.

Reply:
Done